### PR TITLE
added file for addPassword service action and updated changelog

### PR DESCRIPTION
**Pull request by connorlee77**

### CODE REVIEW

File: api-reference/beta/api/passwordCredential_post.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Create owner
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+Use this API to autogenerates and sets a new application [password credential](../resources/passwordcredential.md).
```
Comment (dkershaw10) [inline api-reference/beta/api/passwordCredential_post.md]: autogenerates and sets -> autogenerate and set

File: api-reference/beta/api/passwordCredential_post.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) |  Directory.AccessAsUser.All    |
+|Delegated (personal Microsoft account) | Not supported.    |
+|Application | Application.ReadWrite.OwnedBy and Directory.Read.All, Application.ReadWrite.All and Directory.Read.All |
```
Comment (dkershaw10) [inline api-reference/beta/api/passwordCredential_post.md]: Ah - ouch.  Why is the additional Directory.Read.All permission required?

File: api-reference/beta/api/passwordCredential_post.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+Use this API to autogenerates and sets a new application [password credential](../resources/passwordcredential.md).
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) |  Directory.AccessAsUser.All    |
```
Comment (dkershaw10) [inline api-reference/beta/api/passwordCredential_post.md]: When are we going to have Application.ReadWrite.OwnedBy delegated permission.

File: api-reference/beta/api/passwordCredential_post.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+```http
+POST https://graph.microsoft.com/beta/applications/{id}/addPassword
+Content-type: application/json
+Content-length: 30
+
+{
+  "passwordCredential": {}
+}
+```
+In the request body, supply a JSON representation of [password credential](../resources/passwordcredential.md) object without a specified password value.
```
Comment (dkershaw10) [inline api-reference/beta/api/passwordCredential_post.md]: The property is called `secretText` - should you say without setting `secretText`?

**Comment by Lauragra** [comment]
@connorlee77 , what is the status of this PR? Dan provided review comments that haven't been incorporated. Do you still want to publish this? If so, please resolve all comments and open a new PR from a branch based on master. Otherwise, let's close this one. Thanks.

**Comment by Lauragra** [comment]
@lleonard-msft , Dan had provided feedback on this PR that was never incorporated. Can you look into this please and if we still need to merge this, can you follow up on getting the feedback incorporated? (Likely we'll need a new PR.) Thanks.

**Comment by lleonard-msft** [comment]
@Lauragra - Added to the backlog.

**Comment by Lauragra** [comment]
@davidmu1 , assigning to you to take over for Lance. We have open comments on this PR that haven't been resolved. Can you please investigate and determine whether we want to merge this? Likely we'll need a new PR for the content given that this was opened before the docs migration. @CelesteDG , FYI.

**Comment by davidmu1** [comment]
Suresh is updating this content for GA. Work will be done in a new PR.

**Pull request closed without merging**